review/conftest.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+from review.models import Review
+
+
+@pytest.fixture
+def test_daycare():
```

Comment:
BTW I will recommend removing the `test_` from all fixture's names. it is a fixture, not a test. it'll also make all tests more readable 
(for a different issue/PR though)

Reply:
changed